src/main/java/core/basesyntax/db/FruitsData.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public final class FruitsData {
+    public static final Map<String, Integer> fruitMap = new HashMap<>();
```

Comment:
Maybe it's better to Store Fruits here, as your Fruit model has name and quantity?

Reply:
Yes, and it's already like that

Reply:
FruitDaoImpl
    public void put(Fruit fruit) {
        FruitsData.fruitMap.put(fruit.getName(), fruit.getAmount());
    }

Sorry, i don't know how to quote code here